public/jmap/ajax.d.ts

```diff
@@ -1,6 +1,26 @@
 declare interface JRequestConfig {
-  url: string // The url to request.
-  params?: { [key: string]: any } | string // Body params, that will be passed in the request.
-  accessToken?: string // For request on Jaaz serveur only.
-  includeHeadersInResponse?: boolean // If true will return the header in the response.
+  /**
+   * The url to request.
+   */
+  url: string
+  /**
+   * Body params, that will be passed in the request.
+   */
+  params?: { [key: string]: any } | string
+  /**
+   * For request on Jaaz serveur only.
```

Comment:
For request _**targeting JMap servers**_ only. (on l'ajoute aussi maintenant pour les serveurs JMS et JMC)